jobs/rootfs-builder.jpl

```diff
@@ -103,7 +103,7 @@ node("debos && docker") {
     def docker_image = "${params.DOCKER_BASE}${rootfs_type}"
 
     if (rootfs_type == "chromiumos") {
-        docker_image = "kernelci/cros-sdk"
+        docker_image = "${params.DOCKER_BASE}cros-sdk"
```

Comment:
I don't think this is going to work because the Chrome OS rootfs builds use `kernelci/staging-` as a base and the `cros-sdk` image is only built once for `kernelci/cros-sdk`.

Reply:
Maybe i should change also default prefix for ChromeOS rootfs builds? Cause feature is highly useful to test pending changes.

Reply:
Yes, but I'm not sure how to do this without adding a new Jenkins job parameter. When I've had to do things like this for development I added a patch to my branch and used that with a custom job in Jenkins.